mcp/server.py

```
@@ -761,5 +761,74 @@ async def run_e2e(
+    try:
+        proc = await asyncio.create_subprocess_exec(
+            "psql", dsn, "-t", "-c", "SELECT count(*) FROM prow_job_runs",
+            stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+        )
+        stdout, _ = await asyncio.wait_for(proc.communicate(), timeout=10)
+        count = stdout.decode().strip()
+        if proc.returncode == 0 and int(count) > 0:
+            results.append(f"seed data: OK ({count} job runs)")
+        else:
+            results.append("seed data: EMPTY (run './sippy seed-data --init-database')")
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)